File: python/5073번.py
while True:
    li=list(map(int,input().split()))
    A,B,C=sorted(li,reverse=True)
    if A==0 and B==0 and C==0:
        break
    if A>=B+C:
         print('Invalid')
    elif A==B==C:
        print('Equilateral')
    elif A==B or B==C or A==C:
        print('Isosceles')
    else:
        print('Scalene')

File: java/5073번.java
import java.io.IOException;
import java.util.*;
import java.math.BigInteger;
public class Main {
    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);
        while(true){
            int A=sc.nextInt();
            int B=sc.nextInt();
            int C=sc.nextInt();
            String res = "";
            if(A==0&&B==0&&C==0){break;}
            int[] tmp={A,B,C};
            Arrays.sort(tmp);
            if(tmp[0]+tmp[1]<=tmp[2]){
                res="Invalid";
            }
            else if(A==B&&B==C){
                res="Equilateral";
            }else if(A==B||B==C||A==C){
                res="Isosceles";
            }else{
                res="Scalene";
            }
            System.out.println(res);
        }
    }
}